### ISSUE TITLE
Is there any script to merge splitted data? 

### ISSUE THREAD

**Issue by nedvedj** [kaldi-asr/kaldi#2540]
Hi, I used cleanup/split_long_utterance.sh to split long wavs in my test set. From decode, I got splitted data which can not be compared with original non splitted text (split_long_utterance split only wavs). Is there any script which merge output from decode back to one utterence?

**Comment by vimalmanohar** [comment]
split_long_utterances.sh is for splitting training data using text. It should have split even text. For test data, you might want to use steps/segmentation/detect_speech_activity.sh.
Here is an example recipe on swbd: https://github.com/kaldi-asr/kaldi/blob/master/egs/swbd/s5c/local/run_asr_segmentation.sh

**Comment by nedvedj** [comment]
It generated only text.orig, orig2utt, reco2file_and_channel and segments which define how to devide wavs, but nothing happened with text :-/

Thank you for a link, I will look at it

**Comment by danpovey** [comment]
That segmentation script, steps/segmentation/detect_speech_activity.sh, is intended for when you don't have transcripts.
As for cleanup/split_long_utterance.sh, I'm sure it would be possible to merge the text back to the original utterance, but it's probably not a good idea, as that script may have disacrded some portions of the file that didn't seem to match the transcript, so merging the text back probably isn't a good idea.
You should probably just continue reading and trying things for now.  This is the kind of thing that once you become more proficient with kaldi, you can easily do.  Also see kaldi-asr.org/forums.html for how to ask questions.
